### ISSUE TITLE
Explicit Python 3 Support on pypi

### ISSUE THREAD

**Issue by pachewise** [vertica/vertica-python#206]
Right now, caniusepython3 fails for vertica-python because it doesn't explicitly support py3 on pypi.

would suggest adding the following classifiers (edited, misread the versions from tox.ini):
```python
"Programming Language :: Python :: 2",
"Programming Language :: Python :: 2.7",
"Programming Language :: Python :: 3",
"Programming Language :: Python :: 3.4",
"Programming Language :: Python :: 3.5",
"Programming Language :: Python :: 3.6",
```

**Comment by sitingren** [comment]
Great suggestion! We plan to release a version shortly to exercise the release process. This is a great piece to add. I'll send out a commit soon. Thanks!

**Comment by sitingren** [comment]
Merged fix #224
Verified updates on https://pypi.org/project/vertica-python/
